tests/llms/test_litellm.py

```diff
@@ -152,3 +152,52 @@ def test_chat_model_basic(monkeypatch: MonkeyPatch) -> None:
 def test_metadata() -> None:
     llm = LiteLLM(model="gpt-3.5-turbo")
     assert isinstance(llm.metadata.context_window, int)
+
+# Extra local LiteLLM Tests
+
+# from llama_index.llms import LiteLLM, ChatMessage
+
+# message = ChatMessage(role="user", content="why does LiteLLM love LlamaIndex")
+
+# # # deep infra call
+# os.environ["DEEPINFRA_API_KEY"] = "60UsYw6vmDkYMLVLIZ9vRzHhqOmibo0b" # api key
+# os.environ["OPENAI_API_KEY"] = "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"
```

Comment:
You probably didn't mean to commit your key here... Might need to revoke that

Reply:
oh no dammit !